src/components/add-site.tsx

```diff
@@ -145,6 +145,7 @@ export default function AddSite( { className }: AddSiteProps ) {
 						isDismissible
 						focusOnMount="firstContentElement"
 						onRequestClose={ closeModal }
+						className="max-h-[90%] md:max-h-[90%]"
```

Comment:
I wonder if it's necessary to define the class name `md:max-h-[90%]`. `md` represents a minimum width of 768px ([documentation](https://tailwindcss.com/docs/responsive-design)) and the [minimum window size of the app is 900px](https://github.com/Automattic/studio/blob/4e0d681c98e13dbe357e92e4cfcf16e7731f6881/src/constants.ts#L1), so it will always overlap the height set by `max-h-[90%]`.

Reply:
Oh, ok. My understanding of using the prefix `md` is that the styles associated with `max-h-[90%]` will be applied when the window width is equal to or greater than `768px`. So, in theory, both will always be applied as the window width will be over `768px`.

@katinthehatsite regarding removing the `md:max-h-[90%]`, do you see any difference? I might be overlooking a testing case, but as far as I tested, the modal seems to adapt properly.

Reply:
actually you are right, I can see it is working now 🤔 
I could not make it work before and it was baffling me but perhaps it was related to something in my test environment. I will add a small fix to remove it 👍